plugins/content/caption/search/lib/model/CaptionAssetItem.php

```diff
@@ -155,7 +167,7 @@ public function getDefault()
 	 */
 	public function getUpdatedAt($format = 'Y-m-d H:i:s')
 	{
-		return $this->getAsset()->getUpdatedAt($format);
+		return $this->getAssetNoFilter()->getUpdatedAt($format);
```

Comment:
What about the case that $this->aAsset is null and there is no getCaptionAssetId()?
Please check that getAssetNoFilter has returned a value.